### DISCUSSION TITLE
Left-truncated data with time-varying fitter

### DISCUSSION THREAD

**Discussion by benslack19** [CamDavidsonPilon/lifelines#1577] [Q&A]
Is it safe to assume that `CoxTimeVaryingFitter` supports left truncated data by simply having the first entry for a subject start at one's entry (e.g. `W` in the `CoxPHFitter` model)?

An example table would be like this:

| id | start | stop  | E |
|---|---|---|---|
| 0 | 22 | 25  | 0 |
| 0 | 25 | 27  | 0 |
| 0 | 27 | 30  | 1 |
| 1 | 13 | 15  | 0 |
| 1 | 15 | 25  | 0 |

**Comment by benslack19** [comment]
Any thoughts on this @CamDavidsonPilon ?

**Comment by CamDavidsonPilon** [comment]
Sorry about the delay. Yea, I _believe_ this is correct. However, do some tests!

**Comment by danielwiegand** [comment]
Hi @benslack19, @CamDavidsonPilon, does this mean that left truncation in time-varying survival regression is expressed simply by a subject's lowest `start` value being larger than 0?
